cli/internal/runcache/runcache.go

```diff
@@ -283,10 +284,10 @@ func (tc TaskCache) SaveOutputs(ctx context.Context, logger hclog.Logger, termin
 			terminal.Error(fmt.Sprintf("%s%s", ui.ERROR_PREFIX, color.RedString(" %v", fmt.Errorf("File path cannot be made relative: %w", err))))
 			continue
 		}
-		relativePaths[index] = relativePath
+		relativePaths[index] = fs.UnsafeToAnchoredSystemPath(relativePath)
```

Comment:
This is a known value at this point, but I think this whole loop should be cleaned up. We're doing too much work here.